source/elements/oneDAL/source/data_management/index.rst

```diff
@@ -5,12 +5,140 @@
 Data management
 ===============
 
-The Data Management component includes classes and utilities for data acquisition, initial preprocessing
-and normalization, for data conversion into numeric formats (performed by one of supported Data Sources)
-and for model representation.
+Effective data management is among key constituents of the performance of a data
+analytics application. For |dal_short_name|, effective data management requires
+effectively performing the following operations:
+
+- raw data acquisition
+- filtering and normalization
+- data conversion to a numeric representation
+- data streaming to an algorithm
+
+Key concepts
+============
+
+|dal_short_name| provides a set of customizable interfaces to operate on
+out-of-memory and in-memory data in different usage scenarios, which include
+:term:`batch processing <Batch mode>`, :term:`online processing <Online mode>`,
+and complex scenarios.
+
+Table
+-----
+
+To represent a :term:`dataset` in a numeric :term:`data format`,
+|dal_short_name| introduces a :term:`table` concept - an :term:`immutable
+<Immutability>` tabular view of a :term:`dataset` where table rows represent
+:term:`observations <Observation>` and columns represent :term:`features
+<Feature>`.
+
+.. image:: _static/table.png
+  :width: 400
+  :alt: Table
+
+|dal_short_name| defines a set of table types, each represent data in
+various :term:`data formats <Data format>`. See :ref:`tables` section for more
+details about requirements of table types and table concept.
+
+Data source
```

Comment:
There are multiple data sources, right?

I don't think this section provides a clear explanation of what data source is. 

This content should be more structured. 
After the definition for a data source, I think you need to have something like "data sources allow to:" and a bullet list with operations and explanations for why and when these operations might be needed